lib/extensions/conditional/active_record/base.rb

```diff
@@ -32,6 +32,18 @@ def specific_conditions
     def conditions_satisfied_by?(course_user)
       conditions.all? { |condition| condition.satisfied_by?(course_user) }
     end
+
+    # Unlock the conditional for the given course user.
+    #
+    # @param [CourseUser] _course_user The course user in which the conditional is to unlock for
+    def unlock_for(_course_user)
```

Comment:
:confused:  since when was I a naming genius >_<

`satisfied_for` is actually okay, but it can be rather confusing when we context switch between conditions and conditionals; while `preclude_for` is all right. As for `unlock_for` and `lock_for`, I think it works well in the context of Coursemology, however if you want to generalise this framework into a gem next time (:P) then maybe a more general name would work better. 

If you must force a name out of me, what I can think of is `permit_for`. I cannot think of an alternative for `lock_for` or `precluded_for`.